pkg/controller/perconaservermongodb/version.go

```diff
@@ -234,7 +234,6 @@ func (r *ReconcilePerconaServerMongoDB) getVersionMeta(ctx context.Context, cr *
 		HashicorpVaultEnabled: len(cr.Spec.Secrets.Vault) > 0,
 		ClusterWideEnabled:    len(watchNs) == 0,
 		PMMEnabled:            cr.Spec.PMM.Enabled,
-		BackupsEnabled:        len(cr.Spec.Backup.Storages) > 0,
```

Comment:
why did we change logic for this field?

Reply:
@nmarukovich  please provide more information for us :)